Swiftfin tvOS/Views/LiveTVChannelsView.swift

```diff
@@ -55,7 +55,9 @@ struct LiveTVChannelsView: View {
 		let channel = item.channel
 		if channel.type != "Folder" {
 			Button {
-				self.router.route(to: \.videoPlayer, channel)
+                self.viewModel.fetchVideoPlayerViewModel(item: channel) { playerViewModel in
+                    self.router.route(to: \.videoPlayer, playerViewModel)
```

Comment:
For now, literally copy the current video player and rename it to LiveTVPlayerViewController, copy the overlay and rename it to LiveTVOverlay and have the new view controller use it, make its corresponding SwiftUI LiveTVPlayer view, and fix the coordinators to use that view instead.

It's the beginning of work that we need.

Reply:
Sorry, also add TODO comments at the top as well as descriptions saying what needs to happen next with these views.

Also add a TODO comment above VideoPlayerViewController and createVideoPlayerViewController stating that a new class must be created for LiveTV objects and then consumed by the other LiveTV views.

Reply:
I've duplicated the player and overlay for use with LiveTV. 

Tested playback on a real AppleTV- while playback is successful, I'm getting no audio. So I'll have to investigate further on that.

Reply:
I have to chase further to figure out why, but appears the `selectedAudioStreamIndex` from `VideoPlayerViewModel` is getting set to -1 .

Reply:
Okay, that's expected as it doesn't have a default audio stream. At this point, you may actually have to copy over `createVideoPlayerViewModel()` and make a `createLiveTVViewModel()` method to accommodate for this and setting the correct values.

Eventually a `LiveTVViewModel` will be created and used there.

Reply:
I've duplicated the create viewModel for now. 
But I'm not sure what to adjust to fix audio playback within that method. There is a stereo aac audio track there, but for some reason I don't get any audio playback. 
The same item plays fine with audio in Jellyfin web interface in chrome. Playback from there or the appleTV shows up in the dashboard as a remux. 
Any ideas on what to tweak to try to fix audio playback? Thanks.